src/orderbook.js

```diff
@@ -1197,6 +1336,44 @@ OrderBook.prototype.computeAutobridgedOffers = function() {
   this._offersAutobridged = autobridgeCalculator.calculate();
 };
 
+OrderBook.prototype.computeAutobridgedOffersWrapper = function() {
+  var startTime = Date.now();
+  this.computeAutobridgedOffers();
+  this.mergeDirectAndAutobridgedBooks();
+  var lasted = (Date.now() - startTime);
+  
+  const newMult =
+    ((lasted * 2 / OrderBook.AUTOBRIDGE_CALCULATE_THROTTLE_TIME) << 0) + 1;
```

Comment:
> But my benchmark on latest firefox shows that shifting is 90 times faster.

I get that it is only 3% faster in Firefox and slower in Chrome: http://jsperf.com/math-floor-alternatives

> And, shifting is different operation from Math.floor - shifting make value to be integer, so it clamps it to 32 bits.

All the more reason to not use shifting when you really mean floor?